tuf/tuf.go

```diff
@@ -179,7 +179,7 @@ func (tr *Repo) GetDelegation(role string) (*data.Role, error) {
 // UpdateDelegations updates the appropriate delegations, either adding
 // a new delegation or updating an existing one. If keys are
 // provided, the IDs will be added to the role (if they do not exist
-// there already), and the keys will be added to the targets file.
+// there already), and the keys will be added to the parent role file.
```

Comment:
I originally thought I needed to add a delegations to test AddTarget and RemoveTarget, and I wanted to verify this assumption.  As it turns out, I didn't need it, but doesn't hurt to have the test for it.